packages/@blockprotocol/hook/README.md

```diff
@@ -0,0 +1,120 @@
+## Block Protocol – Hook Service
+
+This package implements the Block Protocol Hraph service for blocks and embedding applications.
+
+To get started:
+
+1.  `yarn add @blockprotocol/hook` or `npm install @blockprotocol/hook`
+1.  Follow the instructions to use the hook service as a [block](#blocks) or an [embedding application](#embedding-applications)
+
+## Blocks
+
+To create a `HookBlockHandler`, pass the constructor an element in your block, along with any callbacks you wish to register to handle incoming messages.
+
+To send a hook message, you call the `hook` function.
+
+```typescript
+import { HookBlockHandler } from "@blockprotocol/hook";
+
+const handler = new HookBlockHandler({ element });
+
+handler.hook({
+  data: {
+    blockId: "hookId",
+    node,
+    type: "text",
+    path: "$.text",
+  },
+});
+```
+
+### React example
+
+For React, we provide a `useHookBlockService` hook, which accepts a `ref` to an element. This will return an object with the shape of `{ hookService: HookBlockHandler | null }` which you can use to send hook messages.
+
+We also provide a `useHook` hook to make sending hook messages easier.
+
+```typescript
+import { useHook } from "@blockprotocol/hook/react";
+
+useHook(hookService, nodeRef, "text", "$.text", (node) => {
+  node.innerText = "hook fallback";
```

Comment:
It'd be helpful to add a comment here explaining that this function will be run if the EA doesn't implement the hook service, or the particular type of hook requested. 

Or mention that above somewhere.

Reply:
https://github.com/blockprotocol/blockprotocol/commit/931e03c0ee74cf4606cc3df0f6d40d57d5c554fc